pkgs/development/tools/csvq/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "csvq";
+  version = "1.17.1";
```

Comment:
Why not use the latest version? ([`1.17.11`](https://github.com/mithrandie/csvq/releases/tag/v1.17.11))

Reply:
> Why not use the latest version? ([`1.17.11`](https://github.com/mithrandie/csvq/releases/tag/v1.17.11))

Typo, fixing.

Reply:
@pinn3 fixed in the latest push.